stable/overwrite/.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: go
+
+go:
+  - 1.7
+
+install:
+  - go get -t ./...
+  - go get golang.org/x/tools/cmd/cover
+  - go get github.com/mattn/goveralls
+
+go_import_path: gopkg.in/dedis/cothority.v1
+
+script:
+  - make test
+
+after_success:
+  - goveralls -coverprofile=profile.cov -service=travis-ci -repotoken $COVERALLS_TOKEN
```

Comment:
Still not sure we need that ! See the [readme](https://github.com/mattn/goveralls#github-integration) for the simple case.

Reply:
I removed the variable but not the reference - I'm quite surprised travis passed anyway...